src/main/java/com/crud/tasks/domain/BadgesDto.java

```diff
@@ -0,0 +1,18 @@
+package com.crud.tasks.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BadgesDto {
+
+    @JsonProperty("votes")
+    private int votes;
+
+    @JsonProperty("attachments")
```

Comment:
To powinno byc attachmentsByType. Nazwy musza dokladnie sie zgadzac w klasie i json'ie, inaczej serializacja nie zadziala poprawnie.